src/main/config/parameter_group_ids.h

```diff
@@ -62,6 +62,7 @@
 #define PG_CHANNEL_RANGE_CONFIG 44
 #define PG_MODE_COLOR_CONFIG 45
 #define PG_SPECIAL_COLOR_CONFIG 46
+#define PG_LED_TYPE 47
```

Comment:
Ideally this should be called PG_LEDSTRIP_CONFIG .. and individual leds something like PG_LEDSTRIP_LEDS or similar
And add LEDSTRIP prefix to PG_COLOR_CONFIG, PG_MODE_COLOR_CONFIG, PG_SPECIAL_COLOR_CONFIG
And rename PG_COLOR_CONFIG to PG_LEDSTRIP_PALETTE

Or maybe use PG_LED prefix for things related to ledstrip (and not onboard leds), then PG_LED_STRIP_CONFIG and PG_LED_CONFIG will make sense ...